nifi-extension-bundles/nifi-workday-bundle/nifi-workday-processors/src/main/java/org/apache/nifi/processors/workday/GetWorkdayReport.java

```diff
@@ -325,6 +326,8 @@ private FlowFile createResponseFlowFile(FlowFile flowfile, ProcessSession sessio
                         responseFlowFile = session.putAttribute(responseFlowFile, CoreAttributes.MIME_TYPE.key(), mimeType.get());
                     }
                 }
+            } else {
+                getLogger().error("Unsuccessful API call. HTTP status code - {}.", statusCode);
```

Comment:
Recommend removing the trailing `.` and changing this to a warning.
```suggestion
                getLogger().warn("Workday API request failed: HTTP {}", statusCode);
```